src/main/java/item/ItemManage.java

```diff
@@ -0,0 +1,238 @@
+        else if (command.equals("unmark")) {
+            mark(content, false);
+        }
+        else if (command.equals("todo")) {
+            addTodo(content);
+        }
+        else if (command.equals("deadline")) {
+            addDeadlines(content);
+        }
+        else if (command.equals("event")) {
+            addEvents(content);
+        }
+        else if (command.equals("list")) {
+            showItemList();
+        }
+        else if (command.equals("deleteAll")) {
+            deleteAll();
+        }
+        else if (command.equals("delete")) {
+            delete(content);
+        }
+        else {
+            throw new RoboastException("Invalid command");
+        }
+
+    }
+
+    public void mark(String content, boolean isDone) {
+        try{
+            int posInt = Integer.parseInt(content);
+            Item item = itemList.get(posInt-1);
+
+            if (isDone){
+                item.setDone(true);
+                System.out.println(LINE);
+                System.out.println("setDone: " + item.getItemName() + "\n" + LINE);
+            }
+            else{
+                item.setDone(false);
+                System.out.println(LINE);
+                System.out.println("unmarked: " + item.getItemName() + "\n" + LINE);
+            }
+        }
+
+        catch (Exception e){
+            if (itemList.isEmpty()) {
+                System.out.println(LINE);
+                System.out.println("Please add an item before marking!");
+                System.out.println("What else do you want to do?");
+                System.out.println(LINE);
+                return;
+            }
+
+            System.out.println(LINE);
+            System.out.println("Incorrect format for command \"setDone\" or \"unmark\"");
+            System.out.println("Correct format for command \"setDone\" is \"setDone {Positive Integer}\", max = " + (itemList.size()));
+            System.out.println("Correct format for command \"unmark\" is \"unmark {Positive Integer}\", max = " + (itemList.size()));
+            System.out.println("What else do you want to do?");
+        }
+    }
+
+    public void addTodo(String content) {
+        if (content.isEmpty()){
+            showAddEmptyError();
+            return;
+        }
+        System.out.println(LINE);
+        Item item = new Todo(content, false);
+        itemList.add(item);
+        System.out.println("added: " + item.getItemName() + "\n" +
+                "You now have " + itemList.size() + " tasks\n" + LINE);
+    }
+
+    public void addDeadlines(String content) {
+        if (content.isEmpty()){
+            showAddEmptyError();
+            return;
+        }
+        System.out.println(LINE);
+        Item item = new Deadline(content, false);
+        itemList.add(item);
+        System.out.println("added: " + item.getItemName() + "\n" +
+                "You now have " + itemList.size() + " tasks\n" + LINE);
+    }
+
+    public void addEvents(String content) {
```

Comment:
The functions refer to events and deadline in plural, but the functions themselves add only 1 event or 1 deadline at a time. Consider naming them `addDeadline` and `addEvent` instead.